olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -492,7 +492,9 @@ spec:
                       runAsNonRoot: true
                       capabilities:
                         drop:
-                          - all
+                          - ALL
```

Comment:
Using `all` will not work. Warnings still appear. Need capital letters for capabilities.